docs/using_eessi/setting_up_environment.md

```diff
@@ -206,6 +206,41 @@ $ module av
 # ...
 ```
 
+??? tip "Environment variables to consider while initializing EESSI"
+
+    ### Source-based Initialization
+
+    | Environment variable | Description |
+    |-----------------------|------------------------|
+    | `EESSI_SILENT` | When set to 1, suppresses non-error output during source-based init; errors still print. |
+    | `EESSI_USE_ARCHDETECT` | When set to 0, disables automatic architecture detection using archdetect; otherwise detection is performed. |
+    | `EESSI_BASIC_ENV` | When set to 1, performs a minimal init only exporting essential  variables. |
+    | `EESSI_CVMFS_REPO_OVERRIDE` | Overrides the default CVMFS repository path (e.g. /cvmfs/software.eessi.io) used by init. |
+    | `EESSI_VERSION_OVERRIDE` | Overrides the default EESSI version subdirectory (e.g. 2023.06) selected by init. |
+    | `EESSI_SOFTWARE_SUBDIR_OVERRIDE` | Overrides detected CPU software subdir (e.g. linux/x86_64/intel/icelake) for the main stack. |
+    | `EESSI_ACCELERATOR_TARGET_OVERRIDE` | Overrides detected accelerator target (e.g. accel/nvidia/cc70) regardless of local hardware. |
+    | `EESSI_ACCEL_SOFTWARE_SUBDIR_OVERRIDE` | Overrides CPU software subdir used specifically for accelerator stack modules. |
+    | `EESSI_MODULE_SUBDIR` | Overrides the module tree subdir to expose (e.g. modules/all or architecture-specific path). |
+    | `EESSI_CUSTOM_MODULEPATH` | Additional colon-separated module paths to prepend to MODULEPATH during module-based init. |
```

Comment:
We really should just get rid of these, they have never even been properly tested. I wouldn't expose them here